plugins/source/gcp/client/multiplex.go

```diff
@@ -13,3 +15,38 @@ func ProjectMultiplex(meta schema.ClientMeta) []schema.ClientMeta {
 	}
 	return l
 }
+
+func OrgMultiplex(meta schema.ClientMeta) []schema.ClientMeta {
+	client := meta.(*Client)
+
+	projectsClient, err := resourcemanager.NewProjectsClient(client.ctx, client.ClientOptions...)
+	if err != nil {
+		client.Logger().Error().Err(err).Msg("OrgMultiplex: Failed to create ProjectsClient")
+		return []schema.ClientMeta{client}
+	}
+
+	dupes := map[string]struct{}{}
+
+	l := make([]schema.ClientMeta, 0, len(client.projects))
+	for _, projectId := range client.projects {
+		resp, err := projectsClient.GetProject(client.ctx, &resourcemanagerpb.GetProjectRequest{
+			Name: "projects/" + projectId,
+		})
```

Comment:
What about retry logic? The default rate limit for getting projects is 600 req/minute... Not sure how fast this API is though. We might also want to document which quota users can raise if they want to increase performance of this section

Reply:
`NewProjectsClient` is already initialized with `client.ClientOptions...`, do we also need to put it in the `GetProject` call? Not sure